packages/flame_behavior_tree/behavior_tree/analysis_options.yaml

```diff
@@ -0,0 +1 @@
+include: package:flame_lint/analysis_options.yaml
```

Comment:
End with a new line (like in the example analysis file)

Reply:
Added a new line here.